src/formats/error.c

```diff
@@ -27,6 +27,8 @@ const char *sd_get_error(enum SD_ERRORCODE error_code) {
             return "File could not be read";
         case SD_FILE_UNLINK_ERROR:
             return "File could not be unlinked";
+        case SD_FAILURE:
```

Comment:
Maybe unsupported might be clearer?

Reply:
I don't know if it matters. The default will be that libpng is enabled; if someone disables it, then it's a known case.

I sort of want to get rid of these codes completely and just abort() on problems :/